frontend/src/pages/Player/Toolbar/DevToolsWindowV2/DevToolsWindowV2.tsx

```diff
@@ -281,33 +296,59 @@ const DevToolsWindowV2: React.FC<
 													}}
 												/>
 											</Box>
-										</label>
+										</Box>
 									</Form>
 
 									{selectedDevToolsTab === Tab.Console ? (
-										<MenuButton
-											divider
-											size="medium"
-											options={Object.values(
-												LogLevel,
-											).map((ll) => ({
-												key: ll,
-												render: ll,
-												variants: LogLevelVariants[
-													ll as keyof typeof LogLevelVariants
-												]
-													? {
-															variant:
-																LogLevelVariants[
-																	ll as keyof typeof LogLevelVariants
-																],
-													  }
-													: undefined,
-											}))}
-											onChange={(ll: string) =>
-												setLogLevel(ll as LogLevel)
-											}
-										/>
+										<>
+											<MenuButton
+												divider
+												size="medium"
+												options={LogLevelValues.map(
+													(level) => ({
+														key: level,
+														render: (
+															<Text case="capital">
+																{level}
+															</Text>
+														),
+													}),
+												)}
+												onChange={(level) => {
+													if (level === 'All') {
+														setLevels([])
+													} else {
+														setLevels([
+															level as LogLevel,
+														])
+													}
+												}}
+											/>
+											<MenuButton
+												divider
+												size="medium"
+												selectedKey={LogSource.Frontend}
```

Comment:
Kind of a hack here -- we need to set the default value of this dropdown to "Frontend". The state variable above is setting this correctly:

https://github.com/highlight/highlight/blob/2c0d451c3155f60a4cbfce46cf645f6c2df3d1c3/frontend/src/pages/Player/Toolbar/DevToolsWindowV2/DevToolsWindowV2.tsx#L80-L82

But it operates as being future proofed to handling multiple log sources (for when we have a proper multiselect component) whereas `<MenuButton />` can only handle one selected value. I could have done something like `selectedKey={sources[0]}` but that would have been kind of brittle. Hence, i'm opting for just duping the behavior and this will be cleaner when we have our multiselect component available.